style.css

```diff
@@ -0,0 +1,284 @@
+html {
+	font-size: 10px;
+	scroll-behavior: smooth;
+	font-family: 'Varela Round', sans-serif;
+	box-sizing: border-box;
+}
+
+nav {
+	position: relative;
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	gap: 30px;
+	background-color: #BD4B4B;
+}
+
+nav div a {
+	padding: 30px;
+}
+
+nav::after {
+ 	position: absolute;
+	bottom: -50px;
+ 	left: 0px;
+ 	width: 100%;
+	content:" ";
+ 	background:
+ 	radial-gradient(circle at 50% 0%, #BD4B4B 25%, #EEEEEE 26%, #EEEEEE 40%);
+ 	background-color: #EEEEEE;
+ 	background-size: 30px 100px;
+ 	height: 50px;
+}
+nav img {
+	width: 175px;
+}
+
+.navigation-links {
+	color: #ffffff;
+	text-decoration: none;
+	font-size: 3rem;
+}
+
+header {
+	background-color: #EEEEEE;
+	height: 90vh;
+	color: #000000;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+	padding: 50px 0;
+}
+
+header p {
+	font-size: 6rem;
+	display: flex;
+	position: relative;
+}
+
+.sparkle-item {
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+	height: 100px;
+	width: 500px;
+	margin: 0;
+}
+
+h1 {
+	font-size: 11rem;
+	margin: 0;
+	align-items: center;
+	text-align: center;
+}
+
+header img {
+	width: 10rem;
+	position: absolute;
+	left: 20px;
+}
+
+.back-to-top-link {
+	display:flex;
+	align-items: center;
+	justify-content: center;
+	padding-bottom: 50px;
+}
+
+header div a {
+	background-color: #EFB7B7;
+	color: #000000;
+	font-size: 2rem;
+	padding: 10px;
+	text-decoration: none;
+	border-radius: 5px;
+	border-style: double;
+	border-width: 3px;
+	position: fixed;
+	bottom: 10px;
+}
+
+.about-section {
+	background-color: #BD4B4B;
+	padding: 50px;
+	height: 100vh;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	line-height: 1.2;
+}
+
+.about-section h2 {
+	padding-left: 30px;
+	font-size: 4rem;
+	color: #ffffff;
+}
+
+.about-section p {
+	padding-left: 30px;
+	padding-bottom: 30px;
+	font-size: 3rem;
+	color: #ffffff;
+	width: 75%;
+	margin: 0;
+}
+
+.about-section a {
+	text-decoration: wavy underline;	
+	font-size: 3rem;
+	color: #ffffff;
+}
+
+.polaroid-photos {
+	width: 500px;
+	height: auto;
+	padding-right: 50px;
+}
+
+.portfolio-section {
+	background-color: #EEEEEE;
+	padding: 50px;
+	min-height: 100vh;
+}
+
+.portfolio-section h2 {
+	display:flex;
+	align-items: center;
+	justify-content: center;
```

Comment:
instead of using flex just to center the text, remove these and use `text-align: center`